sdk_container/src/third_party/coreos-overlay/coreos-base/misc-files/files/oems/ec2

```diff
@@ -0,0 +1,36 @@
+/oem/base/README
```

Comment:
The file supports directories, it's ok to just state `/oem/base`, `/oem/bin`, `/oem/eks` and so on.